couchexport/util.py

```diff
@@ -45,6 +45,10 @@ def clear_attachments(schema_or_doc):
         del schema_or_doc['_attachments']
     if schema_or_doc and 'case_attachments' in schema_or_doc:
         del schema_or_doc['case_attachments']
+    if schema_or_doc:
```

Comment:
yes in concept, though I think at this point `schema_or_doc` is a CommCareCaseAction instance: https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/casexml/apps/case/models.py#L57

Maybe we can do something sketchy like checking for `updated_known_properties`/`updated_unknown_properties` or similar?

Definitely want to limit the likelihood of someone naming a question `"attachments"` and not being able to export it.